tests/get-coverage.yml

```diff
@@ -37,8 +37,18 @@
         state: absent
         path: "{{ coverage_file }}"
 
+    - name: find coverage files to delete
+      find:
+        path: "{{ playbook_dir }}"
```

Comment:
This path seems to be wrong. It needs to be the remote user's home directory.

Reply:
@tyll 
Sir, I have done the required changes pls have a look. And suggest if need any more changes.